src/Microsoft.OData.Core/ODataNestedResourceInfo.cs

```diff
@@ -92,6 +97,36 @@ public Uri AssociationLinkUrl
             }
         }
 
+        /// <summary>Gets or sets the number of items in the resource set.</summary>
+        /// <returns>The number of items in the resource set.</returns>

```

Comment:
Noet: you don't really need the <returns> section for a property -- you cover what it returns in the summary.